emails/tests/fixtures/disabled_mask_for_spam_expected.email

```diff
@@ -208,6 +210,12 @@ lay.firefox.com/fonts/Inter/Inter-Medium.027d14e7d35b.woff2?v=3D3.18) format(=
         }
     </style>
 
+    <div style=3D"display:none !important; color: #ffffff; font-size: 0px; li=
+ne-height: 0; max-height: 0px; max-width: 0px; opacity: 0; overflow: hidden; =
+mso-hide: all;">
+        ???
```

Comment:
`???` means the fluent ID was not found.

Reply:
Whoops, thanks. Overlooked this since, y'know ... it's invisible.